### PR TITLE
Fix mobile view and rework (s)css to use PrimeVue theming or inline PrimeFlex classes

### LINKED ISSUE #7: Quick overview for bar staff
This means the bar committee no longer has to ask for members to spell their name (which given the noise may take quite a bit of time) and also works for members who chose to not have a membership card.

image

### LINKED ISSUE #29: Feature Request: Create mobile layout and styling
Right now the styling on mobile is very wonky. Seeing that people also need to be able to use this site on mobile (looking at the BAC mainly), this should be fixed.

### PR DESCRIPTION
**Pull request by CodeNamedRobin**
Fixes #29

### CODE REVIEW

**Comment by SuperVK** [comment]
Bottom bar is not sticky to the bottom of the screen when content doesn't fill the entire screen
image

File: src/views/BalanceView.vue
Comment (SuperVK) [inline src/views/BalanceView.vue:1]: On line 24 there is still a "severity=danger"

File: src/components/ProductComponent.vue [hunk 1]
```diff
@@ -25,33 +23,33 @@ defineProps({
...
+//}
+//
+//.product-name {
+//  margin: 0;
+//  font-weight: bold;
+//  font-size: 1rem;
+//  overflow: hidden;
+//  text-overflow: ellipsis;
+//  white-space: nowrap;
+//}
```
Comment (SuperVK) [inline src/components/ProductComponent.vue]: Would be nice to remove all of the commented CSS? Unless there is a good reason for it

File: src/components/ProductDialogComponent.vue [hunk 2]
```diff
@@ -55,41 +72,41 @@ const visible = ref(false);
...
+//    }
+//
+//    img {
+//      max-width: 10rem;
+//      max-height: 10rem;
+//      width: 100%;
+//      height: 100%;
+//    }
+//  }
+//}
```
Comment (SuperVK) [inline src/components/ProductDialogComponent.vue]: Same goes here

**Comment by SuperVK** [comment]
LGTM!

File: src/views/LoginView.vue [hunk 3]
```diff
@@ -152,6 +152,35 @@ const toHomeView = () => {
 
 <style scoped lang="scss">
 //TODO Fix the amount of css used related to #14 and #29
```
Comment (SuperVK) [inline src/views/LoginView.vue:164]: This line needs to be removed

File: src/views/PointOfSale/POSEditView.vue [hunk 4]
```diff
@@ -181,81 +178,6 @@ const handleEditPOS = handleSubmit(async (values) => {
 
 <style scoped lang="scss">
 // TODO: Generalize this style and the one from create view
```
Comment (SuperVK) [inline src/views/PointOfSale/POSEditView.vue:180]: This TODO should be fixed with this pull as well right?

File: src/views/SingleUserView.vue [hunk 5]
```diff
@@ -123,35 +130,33 @@ const getUserMutations = async (take: number, skip: number) :
...
-  margin-bottom: 10px;
-  margin-top: 10px;
-}
-
-.content-wrapper {
-  display: flex;
-  flex-direction: column;
-}
+<style scoped lang="scss">
+//form {
```
Comment (SuperVK) [inline src/views/SingleUserView.vue]: Some uncommented code left

File: src/views/TermsOfServiceView.vue [hunk 6]
```diff
@@ -67,7 +67,7 @@ const handleLogout = () => {
 </script>
 
 <style scoped lang="scss">
-@import '../styles/BasePage.css';
+//@import '../styles/BasePage.css';
```
Comment (SuperVK) [inline src/views/TermsOfServiceView.vue]: Same here

File: src/views/TransactionsView.vue [hunk 7]
```diff
@@ -50,5 +50,5 @@ const getUserMutations = async (take: number, skip: number)
 </script>
 
 <style scoped lang="scss">
-@import '../styles/BasePage.css';
+//@import '../styles/BasePage.css';
```
Comment (SuperVK) [inline src/views/TransactionsView.vue]: Same here

File: src/views/PointOfSale/POSEditView.vue (hunk 4 above)
Comment (CodeNamedRobin) [reply src/views/PointOfSale/POSEditView.vue:180]: Nah, I'm still thinking about this as I have no clue how to do this efficiently and correctly.

**Review by JustSamuel — APPROVED** [review]
Mooi

**Pull request merged**